src/paper/tasks.py

```diff
@@ -657,40 +657,6 @@ def celery_update_hot_scores():
         paper.calculate_hot_score()
 
 
-# Executes every 5 minutes
-@periodic_task(
-    run_every=crontab(minute='*/5'),
-    priority=1,
-    options={'queue': f'{APP_ENV}_core_queue'}
-)
-def celery_preload_hub_papers(hub_ids=None):
```

Comment:
this is running periodically every 5 minutes. the decorator @periodic_task specifies that

Reply:
@lightninglu10 sorry, I meant that despite this running periodically, I don't think it does anything productive. Wanted to get confirmation in case I am missing something here